### ISSUE TITLE
SQFormCheckboxGroup select all checkbox needs a key

### ISSUE THREAD

**Issue by 20BBrown14** [SelectQuoteLabs/SQForm#245]
Affected Components: SQFormCheckboxGroup

When building/making updates to this component is seems like the key was forgotten for the "Select All" box on the checkbox group. So if a consumer wants to use the select all functionality they'll be presented with the following console warning. Simply adding a dynamic key based on the group name will resolve this issue.

<img width="562" alt="Screen Shot 2021-05-11 at 3 36 10 PM" src="https://user-images.githubusercontent.com/29528409/117881393-d125e380-b26e-11eb-8c0a-496712ab4ce6.png">

**Comment by SeanGroff** [comment]
I was digging into this a little. I think this is also causing our storybook deploy to fail.

**Comment by 20BBrown14** [comment]
> I think the changes to the SQForm story from that PR may be causing the story test to fail as well

Good call. I updated the relevant PR #246 to resolve those issues as well.
